src/components/DayPicker.jsx

```diff
@@ -436,38 +631,63 @@ export default class DayPicker extends React.Component {
     const transformType = this.isVertical() ? 'translateY' : 'translateX';
     const transformValue = `${transformType}(${translationValue}px)`;
 
+    const shouldFocusDate = !isCalendarMonthGridAnimating && !showKeyboardShortcuts;
+
     return (
-      <div className={dayPickerClassNames} style={dayPickerStyle} >
+      <div className={dayPickerClassNames} style={dayPickerStyle}>
         <OutsideClickHandler onOutsideClick={onOutsideClick}>
-          {!verticalScrollable && this.renderNavigation()}
-
-          <div className="DayPicker__week-headers">
+          <div
```

Comment:
I still think these should be part of the table itself. Seems weird that they're essentially the table headers but not actually part of that markup. It also makes it kind of weird for sighted screen reader users